react-frontend/src/providers/ApolloProviderAsync.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import { ApolloProvider } from 'react-apollo';
+import ApolloClient from 'apollo-boost';
+
+import { AppConfigContext } from './AppConfig';
+
+// The apollo provider implements the strapi configuration
+// by pulling the backend url from a config file. This allows
+// the deployment config file instead of being baked into
+// the build config.
+function ApolloProviderAsync(props) {
+  const config = useContext(AppConfigContext);
+
+  const strapiURL = config['state']['strapiApiUrl'];
+
+  let client = new ApolloClient({
```

Comment:
Why have a const client in apolloClient.js and then initialize one here? Can we re-use the const one everywhere?

Reply:
This may be worth refactoring.  I have a feeling it an issue with appollo client and context was forcing my hand here, but now I'm wondering  if I can work around that.  I'll look into this today.